### DISCUSSION TITLE
Default Android time picker to "type in" mode?

### DISCUSSION THREAD

**Discussion by rcdarwin** [appcelerator/titanium_mobile#12705] [Q&A] [answered]
Is it possible to make an Android time picker default to 'type in' mode vs the clock face mode?

**Comment by jquick-axway** [comment, marked as answer]
Unfortunately no. I'm not aware of any native Java API that will make it default to the text editor side.

Perhaps an alternative solution would be to show a read-only `TextField` instead, and when it receives the focus, show the time picker as a dialog. Now the dialog will default to the clock as well, but I think this type of UI interface looks fine for a dialog. The only issue is doing it this way isn't portable with iOS... although the time picker handling doesn't line up between Android and iOS anyways.
